### PR TITLE
Add line height units example

### PR DESCRIPTION
**Pull request by pepelsbey**
### Description

It adds line height units inline demo and a short description.

### Motivation

It’s a part of the [Firefox release task](https://github.com/mdn/content/issues/29781).

### Additional details

The whole demo looks like this:

Screen Shot 2023-11-09 at 12 25 06

And like this, when the unit is not supported:

Screen Shot 2023-11-09 at 12 25 33

### Related issues and pull requests

Relates to #29781

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 1]
```diff
@@ -265,6 +265,61 @@ However, if you change the `<html>` element's `font-size` in the CSS you will se
 
 {{EmbedGHLiveSample("css-examples/learn/values-units/em-rem.html", '100%', 1100)}}
 
+#### Line height units
+
+`lh` and `rlh` are relative lenghts units similar to `em` and `rem`. The difference between `lh` and `rlh` is that the first one is relative to the line height of the element itself, while the second one is relative to the line height of the root element, usually `<html>`.
+
+Using these units we can precisely align box decoration to the text. In the example below we use `lh` unit to create notepad-like lines using [`repeating-linear-gradient()`](/en-US/docs/Web/CSS/gradient/repeating-linear-gradient). It doesn't matter what's the line height of the text, the lines will always start in the right place.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:272]:
```suggestion
Using these units, we can precisely align box decoration to the text. In this example, we use `lh` unit to create notepad-like lines using [`repeating-linear-gradient()`](/en-US/docs/Web/CSS/gradient/repeating-linear-gradient). It doesn't matter what's the line height of the text, the lines will always start in the right place.
```

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 2]
```diff
@@ -265,6 +265,61 @@ However, if you change the `<html>` element's `font-size` in the CSS you will se
...
+
+```html
+<p style="line-height: 2em">
+  Summer is a time for adventure, and this year was no exception. I had many
+  exciting experiences, but two of my favorites were my trip to the beach and my
+  week at summer camp.
+</p>
+
+<p style="line-height: 4em">
+  At the beach, I spent my days swimming, sunbathing, and building sandcastles.
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:316]:
```suggestion
  At the beach, I spent my days swimming, collecting shells, and building sandcastles.
```
In many cultures, people don't sunbathe. In others, women aren't allowed to. Changing to hopefully be more inclusive.

Comment (estelle) [reply files/en-us/learn/css/building_blocks/values_and_units/index.md:316]: This example will look better if the text lines are a bit shorter. They're wrapping.

**Review by estelle — COMMENTED** [review]
Looks good. Just minor edits suggestions.

File: files/en-us/learn/css/building_blocks/values_and_units/index.md (hunk 2 above)
Comment (pepelsbey) [reply files/en-us/learn/css/building_blocks/values_and_units/index.md]: I made the demo a bit taller so it renders without any scrolling starting from a 968px wide screen.

File: files/en-us/learn/css/building_blocks/values_and_units/index.md [hunk 3]
```diff
@@ -265,6 +265,62 @@ However, if you change the `<html>` element's `font-size` in the CSS you will se
...
+  Summer is a time for adventure, and this year was no exception. I had many
+  exciting experiences, but two of my favorites were my trip to the beach and my
+  week at summer camp.
+</p>
+
+<p style="line-height: 4em">
+  At the beach, I spent my days swimming, collecting shells, and building
+  sandcastles. I also went on a boat ride and saw dolphins swimming alongside
+  us.
+</p>
```
Comment (estelle) [inline files/en-us/learn/css/building_blocks/values_and_units/index.md:319]:
```suggestion
<p style="line-height: 2em">
  Summer is a time for adventure, and this year was no exception.
  I had many exciting experiences, but two of my favorites were my 
  trip to the beach and my week at summer camp.
</p>

<p style="line-height: 4em">
  At the beach, I spent my days swimming, collecting shells, 
  and building sandcastles. I also went on a boat ride and saw 
  dolphins swimming alongside us.
</p>
```
so the lines don't wrap in the code example

**Comment by estelle** [comment]
I guess Prettier likes the long lines. I'll fill an issue.

**Review by estelle — APPROVED** [review]

**Pull request merged**